dev/com.ibm.ws.container.service/resources/com/ibm/ws/container/service/resources/Messages.nlsprops

```diff
@@ -166,4 +166,6 @@ ordered.list.already.contains.CWWKM0482E=CWWKM0482E: WebSphere Application Serve
 ordered.list.already.contains.CWWKM0482E.explanation=CMSG0001
 ordered.list.already.contains.CWWKM0482E.useraction=See the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
 
-
+wrong.annotation.package.CWWKM0483I=CWWKM0483I: The {0} annotation will be ignored on classes in the {1} module. Jakarta EE applications must use the {2} annotation. The affected classes are: {3}
```

Comment:
We are not supposed to use future tense in our messages. Please change "will be" to "is" unless this message requires "will" to be technically correct.

Reply:
done